pkg/apis/certificates/v1alpha1/types.go

```diff
@@ -95,3 +101,34 @@ type CertificateSigningRequestList struct {
 
 	Items []CertificateSigningRequest `json:"items" protobuf:"bytes,2,rep,name=items"`
 }
+
+// KeyUsages specifies valid usage contexts for keys.
+// See: https://tools.ietf.org/html/rfc5280#section-4.2.1.3
+//      https://tools.ietf.org/html/rfc5280#section-4.2.1.12
+type KeyUsage string
+
+const (
+	UsageSigning            KeyUsage = "signing"
+	UsageDigitalSignature            = "digital signature"
```

Comment:
need to repeat KeyUsage type on all lines, they don't inherit (https://play.golang.org/p/j6YuUxGRJk)